docs/CNV/germline-cnv-caller-model.tex

```diff
@@ -0,0 +1,162 @@
+
+\noindent{\em\bf Marginalized warm-up and deterministic annealing ---} The complete inference scheme involves interleaving updates of $q(\mathsf{C})$ and $q(\mathsf{D})$. In practice, we found that the non-convexity of ELBO yielded spurious local minima with poor initialization. To alleviate this issue, we utilize two techniques: (1) We initialize $q(\mathsf{C})$ and $q(\mathsf{D})$ by {\em approximately} marginalizing all discrete RVs, and obtain the first estimate of $q(\mathsf{C})$ from the marginalized model. (2) In the spirit of the deterministic annealing expectation-maximization (DA-EM) algorithm \cite{ueda_deterministic_1998}, we apply entropic regularization to both the continuous and the discrete RVs. In brief, we initially encourage high-entropy variational posteriors by replacing the standard ELBO with $\mathsf{ELBO}(\beta) = \mathbb{E}_{\mathbf{z} \sim q(\mathbf{z})}[\log p(\mathbf{x}, \mathbf{z}) - \beta q(\mathbf{z})]$, where $\beta \geq 1$ is the inverse temperature and is slowly annealed during learning. A similar recipe applies to the variational message passing, where all prior, transition, and emission probabilities are scaled by $\beta^{-1}$ in log space and renormalized. Our PyMC3 implementation of DA-ADVI is provided with GATK \cite{mckenna_genome_2010}.
+
+\section{Benchmarking}\label{sec:bench}
+\begin{figure}
+\includegraphics[width=\columnwidth]{figures/germline-cnv-caller-model/benchmark.pdf}
+\caption{The performance of GATK gCNV, XHMM \cite{fromer_discovery_2012}, and CODEX \cite{jiang_codex:_2015} in detecting CNV events from WES data against a matched WGS callset obtained using Genome STRiP \cite{handsaker_large_2015}.  Performance metrics are reported per WES target.}  
+\label{fig:benchmark}
+\end{figure}
+Finally, we benchmark GATK gCNV using a cohort of whole-exome sequencing (WES) blood-normal samples. We use a manually validated and FDR-controlled callset obtained from matched whole-genome sequencing (WGS) samples using Genome STRiP~\cite{handsaker_large_2015} as the truth callset. To compare, we also include the CNV calls obtained using two popular CNV calling tools, XHMM \cite{fromer_discovery_2012} and CODEX \cite{jiang_codex:_2015}, in the benchmark. The results are shown in Fig.~\ref{sec:bench}. We find that GATK gCNV yields nearly $20\%$ higher sensitivity and $50\%$ higher specificity over all CNV events compared to the other methods.  Although not presented here, benchmarks that stratify by truth variant frequency also show that GATK gCNV performs favorably on common CNV events. More extensive benchmarking results demonstrate even further improvement, primarily resulting from GATK gCNV hyperparmeter optimization using additional WGS truth callsets, and will be presented elsewhere.
```

Comment:
We should at least briefly describe how true positives were matched (some fraction reciprocal overlap?).

Reply:
Also how were true negatives counted?

Reply:
We _collapse_ the WGS calls onto WES targets and assign an integer copy number call to each target by quantization (rounding to nearest integer copy number). Now, counting TP, FP, TN, and FN is trivial: if both WGS and WES say CN=2 on an autosomal target, that's a count for TN, if both say CN=3, that's a count for TP, etc.

Reply:
Great, thanks for confirming.  This is already briefly alluded to in the caption, we can add more detail in the preprint (or update for more recent validations, if appropriate).